### ISSUE TITLE
getMultiFile documents in iOS does not work

### ISSUE THREAD

**Issue by mbenci** [miguelpruivo/flutter_file_picker#288]
It is not possible select multi documents (for example pdf) in iOS, in Android it works correctly.

**Flutter Version details**
[✓] Flutter (Channel stable, v1.17.3, on Mac OS X 10.15.2 19C57, locale it-IT)
    • Flutter version 1.17.3 at /Users/marcobencivenni/flutter
    • Framework revision b041144f83 (5 days ago), 2020-06-04 09:26:11 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/marcobencivenni/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.9.1

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] IntelliJ IDEA Community Edition (version 2020.1.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.45.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.10.1

[✓] Connected device (2 available)
    • iPhone di PlayStore • 23124081297df54b0d4e769a676ed63c052fc31c • ios • iOS 12.4.7
    • iPhone 11 Pro Max   • 67F9E690-A32B-4812-87C3-F0849DDDCBC0     • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-3 (simulator)

**Comment by miguelpruivo** [comment]
Why do you say that’s not possible? You shouldn’t have any issues with that, could you please provide the details of how are you trying to pick or even record a video? Thanks!

**Comment by mbenci** [comment]
Using your example code in iOS as soon as I click a document I come back to the picker page, I can't select more than one file at once.
For media, however, you can select multiple media simultaneously.

<img width="414" alt="Schermata 2020-06-09 alle 12 19 14" src="https://user-images.githubusercontent.com/1177947/84136496-aa9e6d80-aa4b-11ea-9e3b-a29c8da1fa8f.png">
<img width="408" alt="Schermata 2020-06-09 alle 12 19 25" src="https://user-images.githubusercontent.com/1177947/84136494-a96d4080-aa4b-11ea-897f-d0d4a53c9654.png">
<img width="418" alt="Schermata 2020-06-09 alle 12 19 35" src="https://user-images.githubusercontent.com/1177947/84136484-a7a37d00-aa4b-11ea-8d58-85a3d2224685.png">

**Comment by miguelpruivo** [comment]
Hi @mbenci this looks exactly the same as #282 and OP solved it by updating OSX, it looks like it was an issue with Mojave so it seems. Let me know if it worked for you.

**Comment by mbenci** [comment]
I don't have Mojave, I have Catalina 10.15.2 
Anyway I can try to update to the last update and also xCode

**Comment by miguelpruivo** [comment]
@mbenci yes, maybe. Keep me informed.

**Comment by mbenci** [comment]
Everything is now updated but nothing has changed. 
Maybe I'm asking for behavior that can't be achieved in iOS. 
In the video below I show what I get, instead I would like to pick 2 or more PDFs at a time. 
Is it possible or not?

https://drive.google.com/file/d/1I-vSZnwpMxltKc75njPvPociNP_i15dm

**Comment by miguelpruivo** [comment]
@mbenci no, definitely that should possible. Try on your app instead of the example by using `FilePicker.getMultiFile()` and let me know if it worked. Maybe there could be an issue with the example app.

**Comment by mbenci** [comment]
Yesss. In my application it works!!!!
Thanks for your support.
